src/Home.js

```diff
@@ -107,4 +110,21 @@ const styles = StyleSheet.create({
   selectedCheckboxText: {
     color: "white"
   },
+  buttonText: {
+    fontSize: 18,
+    fontWeight: "600",
+    padding: 15,
+    alignSelf: "center",
+    color: "#fff"
+  },
+  button: {
+    position: "absolute",
+    bottom: 30,
+    backgroundColor: "#4696ec",
+    width: 150,
+    alignSelf: "center",
+    borderRadius: 25,
+    shadowOpacity: 0.5,
+    shadowRadius: 5,
```

Comment:
The shadow looks like it is being displayed with a different light source than the existing shadows, maybe use the same `shadowOffset` values that are used in `styles.todoContainer`?

Reply:
Thanks Caleb, updated it with the same `shadowOffset`.